src/components/ButtonsToolbar/index.js

```diff
@@ -21,7 +21,7 @@ const ButtonsToolbar = ({literals, canSubmitStep, canSubmitForm, loginRequired,
               onClick={onNavigatePrevPage}
             ><Literal name="previousText"/></Button>
           </ToolbarList>*/}
-          <ToolbarList>
+          <div className="denhaag-button-group denhaag-button-group--multiple">
```

Comment:
hmm zo te zien hebben we dus ook zelf een custom ToolbarList geïmplementeerd, goed om even te onthouden bij het compleet krijgen van open-forms-sdk zonder fork :)